app/models/request/ChannelOptionalRequest.scala

```diff
@@ -16,12 +16,15 @@
 
 package models.request
 
+import models.ChannelType
+import models.ChannelType.api
+import models.ChannelType.web
 import play.api.mvc.WrappedRequest
 
 trait ChannelOptionalRequest[A] extends WrappedRequest[A] {
 
-  def getChannel: String = headers.get("channel") match {
-    case Some(channel) => channel
-    case None          => "web"
+  def getChannel: ChannelType = headers.get("channel") match {
+    case Some(channel) if channel.equals("api") => api
+    case _                                      => web
```

Comment:
Should there be a case for ncts?

Reply:
not here, no. ncts don't use the channel header